core/src/test/scala/org/scalarelational/AbstractTableSpec.scala

```diff
@@ -546,30 +546,30 @@ trait AbstractTableSpec extends WordSpec with Matchers {
 trait AbstractTestDatastore extends Datastore {
   object test extends Table("test_table") {
     val id = column[Option[Int]]("id", PrimaryKey, AutoIncrement)
-    val name = column[String]("name", Unique, ColumnLength(1024))
+    val name = column[String]("name", Unique, ColumnLength(50))
     val date = column[Option[Timestamp]]("date")
   }
   object suppliers extends Table("SUPPLIER") {
     val id = column[Option[Int]]("SUP_ID", PrimaryKey, AutoIncrement)
-    val name = column[String]("SUP_NAME")
-    val street = column[String]("STREET")
-    val city = column[String]("CITY")
-    val state = column[String]("STATE")
-    val zip = column[String]("ZIP")
+    val name = column[String]("SUP_NAME", ColumnLength(200))
```

Comment:
Then we would need some magic to automatically determine a `ColumnLength` for MariaDB. An an additional check if the sum of the columns (and every other datatype that adds to the row length) exceeds the limit.

This change fixes the tests. And I think we could expect users of scalarelational that want to use MariaDB to know about the row size limitation and introduce a `ColumnLength` in their code.

Setting `DefaultVarCharLength` for MariaDB to say `200` might be an option if we document this, including the implications (set `ColumnLength` if your string is expected to exceed 200 chars, set `ColumnLength` if you use to many `String` columns, ..)